src/dataflow-types/src/client/controller.rs

```diff
@@ -71,6 +91,53 @@ where
                     Box::new(ComputeWrapperClient::new(client, instance));
                 (client, logging)
             }
+            InstanceConfig::Managed { size: _, logging } => {
+                let (orchestrator, storage_compute_addr, storage_workers) =
+                    match (&mut self.orchestrator, &self.storage_config) {
+                        (
+                            Some(orchestrator),
+                            StorageConfig::Remote {
+                                compute_addr,
+                                workers,
+                                ..
+                            },
+                        ) => (orchestrator, compute_addr, workers),
+                        // TODO(benesch): bailing here is too late. Something
+                        // earlier needs to recognize when we can't create managed
+                        // instances.
+                        _ => anyhow::bail!("cannot create managed instances in this configuration"),
+                    };
+                let service = orchestrator
+                    .namespace("compute")
+                    .ensure_service(
+                        &format!("instance-{instance}"),
+                        ServiceConfig {
+                            image: "materialize/dataflowd:unstable".into(),
+                            args: vec![
+                                "--runtime=compute".into(),
+                                format!("--storage-workers={storage_workers}"),
+                                format!("--storage-addr={storage_compute_addr}"),
+                                "0.0.0.0:2101".into(),
```

Comment:
Not sure what the k8s conventions are, but hardcoded IPs/ports seem dangerous to me!